src/middleware.py

```diff
@@ -1,8 +1,22 @@
 from fastapi import Request
+from src.models import SessionLocal, Department
+import contextvars
 
 
-async def print_hello(request: Request, call_next):
-    print("Hello")
+async def start_session(data: dict):
+    session_var = contextvars.ContextVar("session")
+    async with SessionLocal() as session:
+        with session.begin():
+            session_var.set(session)
+            obj = Department(**data)
+            session.add(obj)
+            session.flush()
+            session.refresh(obj)
+            session.expunge_all()
+            return obj
```

Comment:
No-no, remove these lines please, just create and begin the session, you shouldn't create here any records)